packages/rainbowkit/src/components/AsyncImage/AsyncImage.tsx

```diff
@@ -57,36 +56,26 @@ export function AsyncImage({
       testId={testId}
     >
       <Box
-        {...(useAsImage
-          ? { 'aria-hidden': true, as: 'img', src }
-          : isRemoteImage
-            ? // biome-ignore format: design system keys
-              {
+        {...(isRemoteImage
+          ? // biome-ignore format: design system keys
+            {
               "aria-hidden": true,
               as: "img",
               onLoad: setRemoteImageLoaded,
               src,
             }
-            : {
-                backgroundSize: 'cover',
-              })}
+          : { 'aria-hidden': true, as: 'img', src })}
```

Comment:
`esbuild` 20.x has a new way of bundling dataurls with `.svg` and `.png`, which means we can use `img` as of now instead of div and do `backgroundImage: url(...)`

Reply:
`esbuild` 20.x has a new way of bundling dataurl's with `.svg` and `.png`, which means we can use `img` as of now instead of div and do `backgroundImage: url(...)`